Traceback (most recent call last):
  File "/home/john/anaconda3/envs/appstore/lib/python3.10/site-packages/jupyter_cache/executors/utils.py", line 58, in single_nb_execution
    executenb(
  File "/home/john/anaconda3/envs/appstore/lib/python3.10/site-packages/nbclient/client.py", line 1305, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/home/john/anaconda3/envs/appstore/lib/python3.10/site-packages/jupyter_core/utils/__init__.py", line 166, in wrapped
    return loop.run_until_complete(inner)
  File "/home/john/anaconda3/envs/appstore/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/john/anaconda3/envs/appstore/lib/python3.10/site-packages/nbclient/client.py", line 705, in async_execute
    await self.async_execute_cell(
  File "/home/john/anaconda3/envs/appstore/lib/python3.10/site-packages/nbclient/client.py", line 1058, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "/home/john/anaconda3/envs/appstore/lib/python3.10/site-packages/nbclient/client.py", line 914, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import os
import numpy as np
import pandas as pd
from IPython.display import HTML, display_html

from appstore.container import AppstoreContainer

container = AppstoreContainer()
container.init_resources()
pd.set_option("max_colwidth", 400)
------------------


[0;31m---------------------------------------------------------------------------[0m
[0;31mModuleNotFoundError[0m                       Traceback (most recent call last)
Cell [0;32mIn[1], line 6[0m
[1;32m      3[0m [38;5;28;01mimport[39;00m [38;5;21;01mpandas[39;00m [38;5;28;01mas[39;00m [38;5;21;01mpd[39;00m
[1;32m      4[0m [38;5;28;01mfrom[39;00m [38;5;21;01mIPython[39;00m[38;5;21;01m.[39;00m[38;5;21;01mdisplay[39;00m [38;5;28;01mimport[39;00m HTML, display_html
[0;32m----> 6[0m [38;5;28;01mfrom[39;00m [38;5;21;01mappstore[39;00m[38;5;21;01m.[39;00m[38;5;21;01mcontainer[39;00m [38;5;28;01mimport[39;00m AppstoreContainer
[1;32m      8[0m container [38;5;241m=[39m AppstoreContainer()
[1;32m      9[0m container[38;5;241m.[39minit_resources()

[0;31mModuleNotFoundError[0m: No module named 'appstore'

